src/utils/layout/LayoutNode.ts

```diff
@@ -345,4 +352,22 @@ export class LayoutNode<Item extends AnyItem = AnyItem, Type extends ComponentTy
 
     return formDataObj;
   }
+
+  public getRowIndices(): number[] {
+    const rowIndices: number[] = [];
+    if (typeof this.rowIndex !== 'undefined') {
+      rowIndices.splice(0, 0, this.rowIndex);
+    }
+    if (this.parent instanceof LayoutNode) {
+      const parentIndices = this.parent.getRowIndices();
+      if (parentIndices) {
+        rowIndices.splice(0, 0, ...parentIndices);
+      }
+    }
+    return rowIndices;
+  }
+
+  runValidations(options?: IValidationOptions): IValidationObject[] {
+    return runValidationOnNodes([this], options);
+  }
```

Comment:
Nice! Could be worth it adding a comment above this function noting when and why you would/should use this instead of the existing `getValidations()`/`hasValidationMessages()`/etc functions.